candid_diff/src/lib.rs

```diff
@@ -0,0 +1,370 @@
+// Logging:
+#[macro_use]
+extern crate log;
+extern crate env_logger;
```

Comment:
```suggestion
```

Not needed for Rust 2018. If you want to use a macro, you can just `use log::the_macro`;